src/interfaces/messenger.rs

```diff
@@ -7,6 +7,7 @@ use uuid::Uuid;
 pub trait Messenger {
     fn send_packet(&self, conn_id: Uuid, packet: Packet);
     fn broadcast_packet(&self, packet: Packet, source_conn_id: Option<Uuid>, local: bool);
+    fn broadcast_packet_remote(&self, packet: Packet);
```

Comment:
The normal broadcast method doesn't have a way of *only* broadcasting to peers, so I added this. I've got a ticket to refactor broadcasting in general, it's not too pretty right now: https://github.com/DuncanUszkay1/Patchwork/issues/129